modules/robin/src/main/ts/ephox/robin/api/general/TextSearch.ts

```diff
@@ -82,6 +82,7 @@ const expandRight = <E, D>(universe: Universe<E, D>, item: E, offset: number, ra
 // through until the offset left is 0. This is designed to find text node positions that
 // have been fragmented.
 const scanRight = <E, D>(universe: Universe<E, D>, item: E, originalOffset: number): Optional<SpotPoint<E>> => {
+  // MAKE THIS A LOOP
```

Comment:
Why was this added as it doesn't appear to be related to this PR? If it should be added, then can we please make it a TODO (preferably with a Jira) instead?

Reply:
Ignore, you fixed this while I was reviewing :stuck_out_tongue: